src/opencensus.erl

```diff
@@ -108,6 +109,14 @@ http_status_to_trace_status(503) ->
 http_status_to_trace_status(S) ->
     S.
 
+%%--------------------------------------------------------------------
+%% @doc
+%% Returns atom for span kind server to set on trace.
+%% @end
+%%--------------------------------------------------------------------
+-spec span_kind_server() -> span_kind().
+span_kind_server() -> ?SPAN_KIND_SERVER.
+
```

Comment:
need this for elixir lib

Reply:
Cool, can you add it for all the kinds and then i'll merge.

Reply:
👍

Reply:
updated!